utils/crypto/bls/blstest/bls_benchmark_test.go

```diff
@@ -51,7 +53,7 @@ func BenchmarkSign(b *testing.B) {
 }
 
 func BenchmarkVerify(b *testing.B) {
-	privateKey, err := NewSigner()
+	privateKey, err := localsigner.NewSigner()
```

Comment:
One thing that's awkward about this test structure from the perspective of `bls` is that `bls` tests are coupled to the implementation of `localsigner`